src/argilla/client/feedback/schemas/records.py

```diff
@@ -88,6 +88,7 @@ def to_server_payload(self) -> Dict[str, Any]:
         """Method that will be used to create the payload that will be sent to Argilla
         to create a `ResponseSchema` for a `FeedbackRecord`."""
         return {
+            # UUID is not json serializable!!!
```

Comment:
```suggestion
```

Yes, it's not, for the moment we're checking the `user_id` in the Python SDK in `add_records`, but we can review this later to just parse it as a `str` in the `to_server_payload` method